test/server/index.js

```diff
@@ -160,6 +162,31 @@ app.post('/streams/in', function(req, res) {
   })
 });
 
+app.get('/send_cookie', function(req, res) {
```

Comment:
Change the name of this endpoint to "set_cookie", accept a query parameter which shall be the name of the cookie to set.

Reply:
DONE.